rebar.config

```diff
@@ -31,6 +33,7 @@
               ]}.
 
 {plugins, [
+           {grpcbox_plugin, {git, "https://github.com/andymck/grpcbox_plugin.git", {branch, "andymck/ts-master/combined-opts-and-template-changes"}}},
```

Comment:
this should just all get pulled in with sibyl, no?

Reply:
yeah actually you are right, will remove